extensions/Worker.Extensions.EventGrid/src/Worker.Extensions.EventGrid.csproj

```diff
@@ -24,7 +24,7 @@
   </ItemGroup>
```

Comment:
I think the version of this ext should now be `3.4.4`, I think 3.5.0 was a mistake?

https://www.nuget.org/packages/Microsoft.Azure.Functions.Worker.Extensions.EventGrid